addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16963,7 +16963,19 @@ msgctxt "#35216"
 msgid "Support add-ons"
 msgstr ""
 
-#empty strings from id 35217 to 35219
+#. Dialog text asking user to overwrite their savestate file
+#: xbmc/cores/RetroPlayer/RetroPlayer.cpp"
+msgctxt "#35217"
+msgid "The savestate can only be loaded with \"{0:s}\". Erase savestate and continue?"
+msgstr ""
+
+#. Button to erase the savestate
+#: xbmc/cores/RetroPlayer/RetroPlayer.cpp"
+msgctxt "#35218"
+msgid "Erase save"
```

Comment:
Above you use "savestate" twice but here only "save". Is that intended or should we keep it consistent?